packages/devtools_app/lib/src/flutter/split.dart

```diff
@@ -45,22 +50,20 @@ class Split extends StatefulWidget {
   /// Cannot be null.
   final Axis axis;
 
-  /// The child that will be laid out first along [axis].
-  final Widget firstChild;
+  /// The children that will be laid out along [axis].
+  final List<Widget> children;
 
-  /// The child that will be laid out last along [axis].
-  final Widget secondChild;
-
-  /// The fraction of the layout to allocate to [firstChild].
+  /// The fraction of the layout to allocate to each child in [children].
   ///
-  /// [secondChild] will receive a fraction of `1 - initialFirstFraction`.
-  final double initialFirstFraction;
+  /// The index of [initialFractions] corresponds to the child at index of
+  /// [children].
+  final List<double> initialFractions;
 
-  /// The key passed to the divider between [firstChild] and [secondChild].
+  /// The key passed to the divider(s) between each child in [children].
   ///
```

Comment:
indicate that this is the divider between index and index + 1 assuming that is what index represents.

Reply:
done